garak/analyze/report_digest.py

```diff
@@ -231,4 +233,4 @@ def compile_digest(report_path, taxonomy=_config.reporting.taxonomy):
     if len(sys.argv) == 3:
         taxonomy = sys.argv[2]
     digest_content = compile_digest(report_path, taxonomy=taxonomy)
-    print(digest_content.encode("utf-8"))
+    print(digest_content)
```

Comment:
Why remove the encoding here?  I suspect we do not want to revert #732

Reply:
This was intentional - was getting bad output on linux (`b'<!HTML...`. Maybe needs a test.

Reply:
Can you provide a test case that reproduces the `bad output`?

Reply:
Sure,

```
$ python -m garak.analyze.report_digest garak_runs/garak.aaf715cb-d670-4965-a9f5-90072e8520c5.report.jsonl
b'<!DOCTYPE html>\n<html lang="en">\n\n<head>\n<meta name="viewport" content="width=device-width, initial-scale=1">\n<meta charset="UTF-8" />\n<style>\nbody {font-fa
```

If you want to generate an HTML report post-hoc, running a command like this redirected into an html file is I think the primary way of getting that out. It looks like the python behaviors for this differ in linux & windows, where decode returns `bytes` for linux and a (non-unicode) string under windows - this looks like it might be predicated on variations in assumptions/handling round terminal encoding.

Tried this on linux & win and got the same results, with `encode()`d strings yielding a bytes object, though. Do you think this CLI code is currently working as it should on win?